client/components/domains/trademark-claims-notice/index.jsx

```diff
@@ -0,0 +1,563 @@
+					<CompactCard className="trademark-claims-notice__actions">
+						<Button borderless onClick={ this.onReject } disabled={ ! hasScrolledToBottom }>
+							{ translate( 'Choose Another Domain' ) }
+						</Button>
+						<Button primary onClick={ this.onAccept } disabled={ ! hasScrolledToBottom }>
+							{ translate( 'Acknowledge Trademark' ) }
+						</Button>
+					</CompactCard>
+				</div>
+			</div>
+		);
+	};
+
+	renderTrademarkClaimMark = claim => {
+		const { markName } = claim;
+
+		if ( ! markName ) {
+			return;
+		}
+
+		return (
+			<div className="trademark-claims-notice__claim-item" key="mark-name">
+				<span className="trademark-claims-notice__claim-item-label">Mark: </span>
+				<span className="trademark-claims-notice__claim-item-data">{ markName }</span>
+			</div>
+		);
+	};
+
+	renderTrademarkClaimJurisdiction = claim => {
+		const { jurDesc } = claim;
+
+		if ( ! jurDesc ) {
+			return;
+		}
+
+		return (
+			<div className="trademark-claims-notice__claim-item" key="jurisdiction">
+				<span className="trademark-claims-notice__claim-item-label">Jurisdiction: </span>
+				<span className="trademark-claims-notice__claim-item-data">{ jurDesc }</span>
+			</div>
+		);
+	};
+
+	renderTrademarkClaimGoodsAndServices = claim => {
+		let { goodsAndServices } = claim;
+
+		if ( ! goodsAndServices ) {
+			return;
+		}
+
+		if ( ! Array.isArray( goodsAndServices ) ) {
+			goodsAndServices = [ goodsAndServices ];
+		}
+
+		return (
+			<div className="trademark-claims-notice__claim-item" key="goods-and-services">
+				<span className="trademark-claims-notice__claim-item-label">Goods and Services: </span>
+				<ul className="trademark-claims-notice__claim-item-list">
+					{ goodsAndServices.map( ( goods, index ) => (
+						<li className="trademark-claims-notice__claim-item-data" key={ index }>
+							{ goods }
+						</li>
+					) ) }
+				</ul>
+			</div>
+		);
+	};
+
+	renderTrademarkClaimInternationalClassification = claim => {
+		let { classDesc } = claim;
+
+		if ( ! classDesc ) {
+			return;
+		}
+
+		if ( ! Array.isArray( classDesc ) ) {
+			classDesc = [ classDesc ];
+		}
+
+		return (
+			<div className="trademark-claims-notice__claim-item" key="international-classification">
+				<span className="trademark-claims-notice__claim-item-label">
+					International Class of Goods and Services or Equivalent if applicable:{' '}
+				</span>
+				<ul className="trademark-claims-notice__claim-item-list">
+					{ classDesc.map( ( classification, index ) => (
+						<li className="trademark-claims-notice__claim-item-data" key={ index }>
+							{ classification }
+						</li>
+					) ) }
+				</ul>
+			</div>
+		);
+	};
+
+	renderContactInfo = contact => {
+		if ( ! contact ) {
+			return;
+		}
+
+		const { addr } = contact;
+		if ( addr.street && ! Array.isArray( addr.street ) ) {
+			addr.street = [ addr.street ];
+		}
+
+		return (
+			<ul className="trademark-claims-notice__claim-item-list">
+				{ contact.name && (
```

Comment:
Since I hate repetition and was curious, I think we could replace most of these blocks (except for the street one?) with something like:

```
renderClaimItem( contact, 'name', "Name" );
renderClaimItem( contact, 'org', "Organization" );
// etc.

function renderClaimItem( contact, key, label ) {
  if ( contact[ key ] ) {
    return null;
  }

  return (
    <li key={ key }>
      <span className="trademark-claims-notice__claim-item-label">{ label }: </span>
      <span className="trademark-claims-notice__claim-item-data">{ contact[ key ] }</span
    </li>
  );
}
```
or something like that.

Looks like some other functions could benefit from this as well.
Even if in some cases the number of lines would be just slightly less, I'd say it's easier to read code that's written once and then just called even just 2 or 3 times than the same code duplicated 2 or 3 times.